### PR TITLE
Add three new Georgia Tech Faculty

### PR DESCRIPTION
**Pull request by zsoltkira**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -494,6 +494,7 @@ Alexandre Vasconcelos,UFPE,http://www.cin.ufpe.br/~amlv,uTLN0mUAAAAJ
 Alexandre X. Falcão,UNICAMP,http://www.ic.unicamp.br/~afalcao,NOSCHOLARPAGE
 Alexandre Xavier Falcão,UNICAMP,http://www.ic.unicamp.br/~afalcao,NOSCHOLARPAGE
 Alexandre d'Aspremont,Ecole Normale Superieure,http://www.di.ens.fr/~aspremon,JEEwSlQAAAAJ
+Alexandros Daglis,Georgia Institute of Technology,https://parsa.epfl.ch/~daglis/,Mm-dHpMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: already added.

File: csrankings.csv [hunk 2]
```diff
@@ -8618,6 +8619,7 @@ Matthew Fredrikson,Carnegie Mellon University,http://www.cs.cmu.edu/~mfredrik,tM
 Matthew G. Knepley,University at Buffalo,https://cse.buffalo.edu/~knepley,MtzzT2MAAAAJ
 Matthew G. Liptrot,University of Copenhagen,http://ku-dk.academia.edu/MatthewLiptrot,n0NYEY0AAAAJ
 Matthew Green 0001,Johns Hopkins University,https://isi.jhu.edu/~mgreen,X0XWAGkAAAAJ
+Matthew C. Gombolay,Georgia Institute of Technology,http://people.csail.mit.edu/gombolay/,Ihyz20wAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:8622]: Please use a GAtech website.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (zsoltkira) [reply csrankings.csv]: Done.

File: csrankings.csv (hunk 2 above)
Comment (zsoltkira) [reply csrankings.csv:8622]: Done.

**Pull request merged**